src/firefly/js/tables/reducer/TableDataReduer.js

```diff
@@ -18,15 +20,18 @@ export function dataReducer(state={data:{}}, action={}) {
                 return update(root, { [tbl_id] : {selectInfo: {$set: selectInfo}}});
             } else return root;
 
+        case (Cntlr.TABLE_NEW_LOADED)  :
+            if (get(root, [tbl_id, 'tableMeta', 'Loading-Status'], 'COMPLETED') !== 'COMPLETED') {
+                return update(root, { [tbl_id] : {tableMeta: {['Loading-Status']:  {$set: 'COMPLETED'}}}});
```

Comment:
That is a good idea, we still us it but put in into some simple wrappers with lodash type syntax that we all like and seems clear.